### PR TITLE
feat(SRVKP-3566): openshift-pipelines performance CI jobs

### PR DESCRIPTION
**Pull request by jhutar**

### CODE REVIEW

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-tests-main-ocp-pipelines-load-test-ci-test

File: ci-operator/config/openshift-pipelines-test/openshift_pipelines_ci/openshift-pipelines-test-openshift_pipelines_ci-tests-main.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+tests:
+- as: ocp-pipelines-load-test-ci-test
+  steps:
+    cluster_profile: aws
+    env:
+      SCENARIO: test
+    workflow: openshift-pipelines-test-load-test
+  timeout: 8h0m0s
+zz_generated_metadata:
+  branch: tests-main
```
[the remark this answers is not in the document]
Comment (jhutar) [reply ci-operator/config/openshift-pipelines-test/openshift_pipelines_ci/openshift-pipelines-test-openshift_pipelines_ci-tests-main.yaml:46]: Again? For some reason `make update` keeps putting it there. Thank you!

File: ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      - chain: ipi-aws-pre
+    test:
+      - ref: openshift-pipelines-test-load-test
+        timeout: 8h0m0s
+        resources:
+          requests:
+            cpu: 3000m
+            memory: 6Gi
+    post:
+      - ref: redhat-appstudio-gather
```
[the remark this answers is not in the document]
Comment (jhutar) [reply ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-workflow.yaml:18]: Oh, I see. This gathers `Applications` and such. There is a different step for e.g. `Deployments`.

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-tests-main-ocp-pipelines-load-test-ci-test

**Comment by pmacik** [comment]
@jhutar you need to rename
`ci-operator/config/openshift-pipelines-test/openshift_pipelines_ci/openshift-pipelines-test-openshift_pipelines_ci-tests-main.yaml`
 -> 
`ci-operator/config/openshift-pipelines-test/openshift_pipelines_ci/openshift-pipelines-test-openshift_pipelines_ci-main.yaml`

and then run `make update` or `CONTAINER_ENGINE=podman make update`

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-tests-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

File: ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-commands.sh [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+if [[ -s "$BYOC_KUBECONFIG" ]]; then
+    echo -e "byoc kubeconfig exists!"
+else
+    echo "Kubeconfig not exists in $BYOC_KUBECONFIG... Aborting job"
+    exit 1
+fi
+
+cd "$(mktemp -d)"
+
+git clone --branch main "https://github.com/openshift-pipelines-test/openshift_pipelines_ci.git"
```
Comment (pmacik) [inline ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-commands.sh:54]:
```suggestion
git clone --branch main "https://github.com/openshift-pipelines-test/openshift_pipelines_ci.git" .
```

Comment (pmacik) [reply ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-commands.sh:54]: You want to checkout the repo into the current directory (created and entered in by the above command)

Comment (jhutar) [reply ci-operator/step-registry/openshift-pipelines-test/load-test/openshift-pipelines-test-load-test-commands.sh:54]: Yeah :-/ I added `cd openshift_pipelines_ci`

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-images

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-test-openshift_pipelines_ci-main-ocp-pipelines-load-test-ci-test

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-main-scaling-pipelines

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-main-scaling-pipelines

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines

**Review by khrm — COMMENTED** [review]
/lgtm

**Comment by jhutar** [comment]
/pj-rehearse pull-ci-openshift-pipelines-performance-master-scaling-pipelines

**Review by khrm — COMMENTED** [review]
/lgtm

**Pull request merged**